| |Slug |Part Name |Description |Part Type |Length |Modified at |Compatible |Used in |
|----|----|----|----|----|----|----|----|----|
|BBa_B0030 |3e30ad4f-5360-49f7-bda4-60929b0f2971 |RBS.1 (strong) -- modified from R. Weiss |Strong RBS based on Ron Weiss thesis. Strength is considered relative to <bb_part>BBa_B0031</bb_part>, <bb_part>BBa_B0032</bb_part>, <bb_part>BBa_B0033</bb_part>.\n |RBS |15 |2021-09-08T20:22:43.000Z |RFC10 RFC1000 |1028 |
|BBa_I742151 |658f0d8b-f0c7-4a36-950e-4b6c5b799e04 |crtE (geranylgeranyl pyrophosphate synthase) coding sequence. |crtE (geranylgeranyl pyrophosphate synthase) from Pantoea ananatis (formerly Erwinia uredovora) DSMZ 30080 (ATCC 19321). Accession: D90087. Part of the carotenoid biosynthesis parthway. |Coding |912 |2021-09-08T20:22:58.000Z |RFC10 RFC1000 |28 |
|BBa_J18920 |e126bd39-8bc6-4af4-9472-32dbcb75ab4a |2aa GS linker |Highly flexible 2 aa linker. Translates to GS. Codon-optimal for E. coli, yeast, mammalian. |Protein_Domain |6 |2021-09-08T20:23:13.000Z |RFC10 RFC1000 |0 |
|BBa_J61100 |b6c9aa1c-f754-440c-b59f-7eff3d4d6005 |Ribosome Binding Site Family Member |{{JCA_Arkin_RBSFamily}} |RBS |12 |2021-09-08T20:23:20.000Z |RFC10 RFC1000 |91 |
|BBa_K1151001 |1b3d3f14-6a9c-49da-be3b-91a79c745117 |Histidine-rich metal-binding protein |So called for emphasize its origins in Helicobacter pylori and its avidity for nickel. Consisting of 60 aminoacids, the protein is rich in histidine (28 residues, 46.7 %) and contains short repeating motifs, it exists as an equilibration of multimeric forms in solution, with 20-mers (approx. 136 kDa) being the predominant species. Hpn can bind tightly and reversibly up to five Ni2+ ions per each monomer of 7 kDa in a pH-dependent manner (pH 7.4 ). In H. pylori play an important role in storing nickel required to the survival of the bacterium. |Coding |182 |2021-09-08T20:23:43.000Z |RFC10 RFC1000 |0 |
|BBa_K118002 |725d8879-394a-4ab0-9c94-87406dfdc35c |crtB coding sequence encoding phytoene synthase |This is the coding sequence of crtB from Pantoea ananatis (formerly Erwinia uredovora) (Accession number D90087). It encodes phytoene synthase, part of the carotenoid biosynthesis pathway, which converts geranylgeranyl diphosphate to phytoene.    Reference: Misawa,N., Nakagawa,M., Kobayashi,K., Yamano,S., Izawa,Y., Nakamura,K. and Harashima,K. Elucidation of the Erwinia uredovora carotenoid biosynthetic pathway by functional analysis of gene products expressed in Escherichia coli. J. Bacteriol. 172 (12), 6704-6712 (1990) |Coding |933 |2021-09-08T20:23:46.000Z |RFC10 RFC1000 |27 |
|BBa_K118003 |b558167c-534b-4267-9920-2a8f0b73a68b |crtI coding sequence encoding phytoene dehydrogenase |This is the coding sequence of crtI from Pantoea ananatis (formerly Erwinia uredovora) (Accession number D90087). It encodes phytoene dehydrogenase, part of the carotenoid biosynthesis pathway, which converts Phytoene to Lycopene. |Coding |1482 |2021-09-08T20:23:46.000Z |RFC10 RFC1000 |26 |
|BBa_K118008 |ec15658e-b8f9-44f6-949f-944837376858 |crtY coding sequence encoding lycopene B-cyclase |This is the coding sequence of crtY from Pantoea ananatis (formerly Erwinia uredovora) (Accession number D90087). It encodes lycopene B-cyclase, part of the carotenoid biosynthesis pathway, which converts lycopene to B-carotene. |Coding |1152 |2021-09-08T20:23:46.000Z |RFC10 RFC1000 |16 |
|BBa_K2150031 |dc2223ad-cf2b-44b4-8e11-e9412d57ade9 |T7 promoter without RBS |This part is the T7 promoter without RBS. It has been reported that T7 promoter doesn't work with E.coli RNA polymerase, but only with T7 RNA polymerase(T7RNAP). However, we observed leakage of it in the absence of T7RNAP. |Regulatory |20 |2021-09-08T20:24:56.000Z |RFC10 RFC1000 |0 |
|BBa_K2306003 |ba6d2f74-5ac4-4711-94a4-5e691c345beb |Secretory-abundant heat soluble protein 33020 (SAHS 33020) |This Biobrick features the gene sequence that encodes for the production of the tardigrade intrisically disordered protein (TDP) "SAHS 68234". This is one of the several heat soluble proteins found in the tardigrades capable of protecting other fragile biological parts upon desiccation by generating a crystal matrix. Therefore, this tardigrade protein can be combined with other perishable biological material (e.g. proteins) to increase its stability after dehydration. |Coding |509 |2021-09-08T20:25:06.000Z |RFC10 |0 |
|BBa_K3457006 |446a23fb-0b65-404e-aa15-d9e87a050856 |lac operator |Olac |Regulatory |25 |2021-09-08T20:26:39.000Z |RFC10 RFC1000 |0 |
|BBa_K3648010 |073ce20c-d54b-48b9-b51f-37a08006b486 |T7 terminator |It is a terminator on the pET-29(b) as a "stop" sign in RNA translation. |Terminator |48 |2021-09-08T20:26:52.000Z |RFC10 RFC1000 |0 |
|BBa_K4162001 |eb325d78-4947-490a-8762-98c87f1b16ea |StayGold = (n2)oxStayGold(c4)v2.0 |The ybbO gene was confirmed to have a retinol dehydrogenase activity that is able to convert retinal into retinol, a property that could be used for the    selective production of retinol from E. coli. |Coding |711 |2022-10-07T01:47:06.000Z |RFC10 |0 |
|BBa_K4162003 |501c58e3-497b-4641-b116-84b4ee684f99 | ybbO coding for retinol dehydrogenase, codon optimized for E. coli |The ybbO gene was confirmed to have a retinol dehydrogenase activity that is able to convert retinal into retinol, a property that could be used for the selective production of retinol from E. coli. |Coding |810 |2022-10-07T01:59:31.000Z |RFC10 RFC1000 |0 |
|BBa_K4162004 |b4a96787-1bb4-475c-b35c-862529f7b5a9 |Gene coding for BCMO from uncultured marine bacteria |Retinal can be produced by introducing the &#946;-carotene mono(di)oxygenase (BCM(D)O) gene into recombinant E. coli that produces &#946;-carotene. |Coding |828 |2022-09-26T08:56:42.000Z |RFC10 RFC1000 |0 |
|BBa_K4162005 |6ef83b5c-d117-4c51-89c6-7a42ac8eca7c |Hammerhead ribozyme |Ribozymes are non-coding RNAs that can catalyze specific biochemical reactions without the assistance of proteins, involved in a variety of vital cellular reactions. Ribozymes can be divided into RNA metalloenzymes and self-cleaving ribozymes. Studies have shown that self-cleaving ribozymes adopt a general acid-base catalytic mechanism to perform site-specific phosphodiester bond cleavage. Until now, only ten families of self-cleaving ribozymes have been discovered, and hammerhead ribozyme is one of them. By inserting the DNA sequence of hammerhead ribozyme between two coding sequences, mRNA transcribed by this gene will perform self cleavage at the ribozyme site, so that a long piece of mRNA will be cut into segments to prevent the translation process of various coding sequences from interfering with each other. |RNA |57 |2023-06-16T07:49:21.000Z |RFC10 RFC1000 |0 |
|BBa_K4162006 |b08abf10-171c-4f0a-8017-47e8211ba923 |T7_RBS |T7 RBS derived from the T7 phage genome. |RBS |17 |2022-10-07T02:01:50.000Z |RFC10 RFC1000 |0 |
|BBa_K4162009 |acb6cbd9-a9c6-4e9f-a64e-5c2cd8b80bbe |ribozyme + B0_RBS + crtE |Coding sequence of crtE under hammerhead ribozyme, with a strong ribosome binding site(BBa_B0030) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |991 |2022-10-13T00:38:01.000Z |RFC10 RFC1000 |0 |
|BBa_K4162010 |41c47c1a-dc7a-4c34-9b29-935f8b255faa |ribozyme + T7_RBS + crtE |Coding sequence of crtE under hammerhead ribozyme, with T7 RBS(BBa_K4162006) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |993 |2022-10-07T02:05:22.000Z |RFC10 RFC1000 |0 |
|BBa_K4162011 |a24160dd-d8c0-42c6-9c62-7dba86bd787d | ribozyme + J6_RBS + crtE |Coding sequence of crtE under hammerhead ribozyme, with a weak ribosome binding site(BBa_J61100) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |981 |2022-10-07T02:06:32.000Z |RFC10 RFC1000 |0 |
|BBa_K4162012 |82d08bc9-9c99-484a-ae41-fe8d0ec139be |ribozyme + B0_RBS + crtB |Coding sequence of crtB under hammerhead ribozyme, with a strong ribosome binding site(BBa_B0030) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |1012 |2022-10-07T02:07:24.000Z |RFC10 RFC1000 |0 |
|BBa_K4162013 |bb7b1c73-7d2f-4731-b61d-dec443d6855c |ribozyme + T7_RBS + crtB |Coding sequence of crtB under hammerhead ribozyme, with T7 RBS(BBa_K4162006) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |1014 |2022-10-07T02:07:53.000Z |RFC10 RFC1000 |0 |
|BBa_K4162014 |4aa731cc-eccb-44e5-8a05-93daf6b21faf |ribozyme + J6_RBS + crtB |Coding sequence of crtB under hammerhead ribozyme, with a weak ribosome binding site(BBa_J61100) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |1002 |2022-10-07T02:10:42.000Z |RFC10 RFC1000 |0 |
|BBa_K4162016 |40fe3b42-c16e-4d4b-b8ca-1a76489ac1fc |ribozyme + T7_RBS + crtI |Coding sequence of crtI under hammerhead ribozyme, with T7 RBS(BBa_K4162006) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |1563 |2022-10-07T02:13:27.000Z |RFC10 RFC1000 |0 |
|BBa_K4162019 |173d275c-6b64-442a-a077-b76291b16800 |ribozyme + T7_RBS + crtY |Coding sequence of crtY under hammerhead ribozyme, with T7 RBS(BBa_K4162006) between them. This part can be used to assemble polycistrons in Escherichia coli. |DNA |1233 |2022-10-07T02:23:24.000Z |RFC10 RFC1000 |0 |
|BBa_K4162021 |262a488e-e5cf-4769-b0b5-c636bde3b345 |ribozyme+RBS+CDS module: crtIYEB |This biobrick was created through standard biobrick assembly of K4162016(ribozyme+RBS+crtE), K4162019(ribozyme+RBS+crtB), K4162010(ribozyme+RBS+crtI) and K4162013(ribozyme+RBS+crtY). These genes are a part of the carotenoid biosynthesis pathway and together, this biobrick converts farnesyl pyrophosphate to beta-carotene. In this part, crtIYEB are separated by hammerhead ribozyme, which can perform self-cleaving. Therefore, the long piece of mRNA it transcribes into will be spontaneously separated into four segments, which correspond to crtI, Y, E and B repectively. The translation process of these four enzymes is independent of each other, and the proportion of the expression products of four enzymes can be accurately adjusted by monitoring the intensity of RBS. |DNA |4896 |2022-10-07T06:35:11.000Z |RFC10 RFC1000 |0 |
|BBa_K4655016 |a8680443-ebf8-4933-8472-bce65dc91b1c |Mutated pR |Mutated version lux promoter of BBa_R0062 reported by Valencia_UPV iGEM 2018. This was reported as Mut9 in Valencia_UPV iGEM 2018 characterization. |Regulatory |65 |2023-10-11T18:18:14.000Z |RFC10 RFC1000 |0 |
|BBa_K4765020 |b79c63fc-a76f-4125-8fb2-bfe4dea48732 |Twister P1 ribozyme, from 10.1080/15476286.2022.2123640 |The twister ribozyme is an RNA molecule that is capable of self-cleavage at specific sites. |RNA |44 |2023-10-06T22:32:55.000Z |RFC10 RFC1000 |0 |
|BBa_K4765021 |25f44d48-32d8-440f-a666-cfa77a98a99f |stem-loop 1, from 10.1021/acssynbio.2c00416 |The stem-loop is a piece of RNA that forms the stem loop structure, which is located at the 3' end of the mono-cistron cut by ribozyme, preventing the degradation of mono-cistron mRNA. |RNA |20 |2023-10-07T22:06:22.000Z |RFC10 RFC1000 |0 |
|BBa_K4765022 |a522ff03-af39-4317-beea-7260cc7d93dd |mScarlet |Strong bacterial ribosome binding site |Coding |699 |2023-10-06T22:35:36.000Z |RFC1000 |0 |
|BBa_K4765111 |74eebf57-611f-43a2-ae8b-8e5cc608b9bb |Twister P1  + T7_RBS + AnAFP + stem-loop |ribozyme+T7_RBS+AnAFP+stem-loop |DNA |693 |2023-10-09T21:18:54.000Z |RFC10 |0 |
|BBa_K4765117 |01011581-614c-401e-8cf0-77d7aabc8b22 |ribozyme connected: H. ex mtSSB + SAHS 33020 |ribozyme+RBS+CDS+stem-loop module:  H. ex mtSSB+SAHS 33020 |DNA |1292 |2023-10-11T04:27:02.000Z |RFC10 |0 |
|BBa_K4765126 |2d4f3647-306f-4d13-ae45-c96843e05598 |ribozyme connected: H. ex mtSSB + SAHS 33020 + AnAFP |ribozyme connected: H. ex mtSSB + SAHS 33020 + AnAFP |DNA |1985 |2023-10-11T04:43:57.000Z |RFC10 |0 |
|BBa_K4767017 |0e8edee1-3a3a-4e36-b7c1-5b7de87a2fa5 |Pbad-RBS-luxI-TT-Ptac-luxR-TT-PluxI-gfp-TT |Uses a factor Plux&#8544;(BBa_R0062) ,strong RBS(BBa_J34801), lux&#8544;(luxI C0061) ,gfP coding region(BBa_E0040), Ptac(M31370), luxR(BBa_C0062)and Pbad(Ara K808000). This part is an easy BioBrick. |Miscellaneous |3941 |2023-10-08T03:43:24.000Z |RFC10 |0 |
|BBa_K5115058 |2cbbda7d-04a6-4631-bf3b-642fe3d6328a |hoxF fused with EP |1 |Miscellaneous |2662 |2024-09-27T09:52:53.000Z |RFC10 |0 |
|BBa_K5115059 |d12c04c4-5c13-4067-8b92-7c5d207c11d5 |hoxU fused with EP |1 |Miscellaneous |1561 |2024-09-27T09:53:27.000Z |RFC10 |0 |
|BBa_K5115086 |ac8e1936-6d2e-4077-9a4f-7affb55518f4 |NixA-F1v |1 |Miscellaneous |1288 |2024-09-30T14:54:44.000Z |RFC10 RFC1000 |0 |
|BBa_K5115087 |fe2c5499-7337-4711-bce5-a2eada8bf19a |F1v-NixA |1 |Miscellaneous |1288 |2024-09-29T10:51:25.000Z |RFC10 RFC1000 |0 |
|BBa_K5441013 |3061cbf2-e6c5-4cfd-8343-88aa38c2357c |BAX |The bax gene is a crucial component in the study of apoptosis, or programmed cell death. Bax encodes a pro-apoptotic protein that promotes cell death in response to various stress signals, playing a significant role in regulating the balance between cell survival and death.        Incorporating the bax gene can help researchers investigate mechanisms of apoptosis in different cell types. By placing bax under the control of a suitable promoter in your plasmid, you can induce its expression and assess its effects on cell viability and death, through ways such as the MTT cell viability assay or ELISA kits. This is particularly valuable for studying cancer biology, as many cancer cells evade apoptosis, contributing to tumor growth and resistance to therapy. |Coding |576 |2024-10-02T09:34:14.000Z |RFC10 RFC1000 |0 |
|BBa_250R9OVR |a239e354-0469-43a3-987b-ff12e3dadc40 |IME1 |Induce meiosis |Coding |1083 |2025-09-13T15:28:07.607Z |RFC10 RFC1000 |0 |
|BBa_K2753051 |4f9b9f3b-633c-47f4-ba95-c7a3bdf1abf1 |tENO1 |a terminator described in Lee (2015) 什么强度 |Terminator |242 |2021-09-08T20:25:40.000Z |RFC10 RFC1000 |2 |
|BBa_25VHXKNL |95f1ea39-9d54-4a8b-9caf-8c0bdbba0914 |ASH1 AIpro |Promoter of ASH1 that can initiate the transcription of modified mCherry(BBa\_25TQG9WZ), predicted by DeePromClass. |Regulatory |2463 |2025-09-13T09:12:37.997Z |RFC10 RFC1000 |1 |
|BBa_25TQG9WZ |dfd14d21-97dc-4618-b0d3-79258c64b7fa |modified mCherry |Fluorescent timer based on mCherry whose color changes over time. At 37 °C, the maxima of the blue fluorescence are observed at 0.25h while the half-maxima of the red fluorescence are reached at 7.1h. |Coding |711 |2025-09-13T10:23:58.163Z |RFC10 RFC1000 |1 |
|BBa_25AIDL8P |d7562128-d208-45a2-b93d-9f4fc83abe42 |ASH1 3'UTR |Guide mRNA to locate in the daughter cells, as descibed in xxxx (year). |Miscellaneous (Deprecated) |250 |2025-09-09T14:51:25.396Z |RFC10 RFC1000 |1 |
|BBa_25PI44VT |d1ff6f30-4531-4c0f-aa10-5e840767f309 |a-Enk |M-Enk with &alpha;-factor secretion signal. |Coding |341 |2025-08-25T12:46:40.746Z |RFC10 |0 |
|BBa_J433100 |d9f02ebe-6ef6-4f55-8e32-a54cdd5e5a04 |KanR-ColE1 |Asimov KanR ColE1 Base Plasmid Backbone https://网址 During Golden Gate with BsaI, it will contribute Kan antibiotic resistance to the final construct. |Plasmid_Backbone |1755 |2024-03-01T10:37:30.000Z |RFC10 RFC1000 |0 |
|BBa_25YPQSK9 |d4bfd000-e3c5-4300-ac04-26e65708bbf4 |AmpR-ColE1 |In the YTK system, AmpR–ColE1 plasmids act as donor vectors, maintaining parts in *E. coli*. During Golden Gate with BsaI, it will contribute Amp antibiotic resistance to the final construct. |Plasmid_Backbone |1850 |2025-09-14T02:25:30.545Z |RFC10 RFC1000 |0 |
|BBa_25TYRLM9 |8a82b32e-db66-4d70-b71c-66bf782abca9 |EMSfp569 |Optimized mScarlet to resist EMS. |Coding |705 |2025-08-25T14:45:50.922Z |RFC10 RFC1000 |0 |
|BBa_25IB5O7X |35d6ff9a-7255-4793-98b5-ec903f960951 |EMSfp499 |Optimized mSG to resist EMS. |Coding |681 |2025-08-25T14:47:55.396Z |RFC10 RFC1000 |0 |
|BBa_25F6RD26 |71784bf0-4d39-4f51-9bee-981a8764250a |EMSfp383 |Optimized eBFP2 to resist EMS. |Coding |720 |2025-08-25T14:49:47.870Z |RFC10 RFC1000 |0 |
|BBa_25GARG3E |b5d5e1d6-cb6c-41c2-a42a-fce74c644b82 |EMSfp642 |Optimized smURFP to resist EMS. |Coding |402 |2025-08-25T14:52:16.694Z |RFC10 RFC1000 |0 |
|BBa_25M2Z9H7 |5dfa7966-37e5-4b68-aa95-6751772e278c |EMSfp399 |Optimized Bluebonnet2 to resist EMS. |Coding |705 |2025-08-25T14:54:03.599Z |RFC10 RFC1000 |0 |
|BBa_2599SI53 |dd48e4c8-8fe9-4aba-a07d-f45c2551a30f |EMSfp643 |Optimized miRFP670-2 to resist EMS. |Coding |936 |2025-08-25T14:58:36.683Z |RFC10 RFC1000 |0 |
|BBa_25XKAUNH |67f477dd-1632-4d58-96ac-25d9699a5fb1 |HR5'_ERG5_chr13L |Knock out ERG5 when used together with HR3'\_ERG5\_chr13R ([BBa\_25E9K479](https://registry.igem.org/parts/bba-25e9k479)), block ergosterol production. |Miscellaneous (Deprecated) |500 |2025-09-13T15:27:03.429Z |RFC10 RFC1000 |0 |
|BBa_25E9K479 |fb9ab746-5948-4285-be65-f2bf0506947d |HR3'_ERG5_chr13R |Knock out ERG5 when used with HR5'\_ERG5\_chr13L ([BBa\_25XKAUNH](https://registry.igem.org/parts/bba-25xkaunh)), block ergosterol production. |Miscellaneous (Deprecated) |500 |2025-09-13T15:25:55.634Z |RFC10 RFC1000 |0 |
|BBa_25U7DH3R |8b4aac2a-f632-4bb9-af36-5253345a8898 |HR5'_ERG6_chr13L |Knock out ERG6 when used together with HR3'\_ERG6\_chr13R ([BBa\_25O1ZVOU](https://registry.igem.org/parts/bba-25o1zvou)), block ergosterol production. |Miscellaneous (Deprecated) |500 |2025-09-14T08:29:56.164Z |RFC10 RFC1000 |0 |
|BBa_25O1ZVOU |04ade6a1-c7ab-4a5b-bbc0-138c287793ff |HR3'_ERG6_chr13R |Knock out ERG6 when used together with HR5'\_ERG6\_chr13L ([BBa\_25U7DH3R](https://registry.igem.org/parts/bba-25u7dh3r)), block ergosterol production. |Miscellaneous (Deprecated) |500 |2025-09-14T07:34:55.097Z |RFC10 RFC1000 |0 |
|BBa_25MIG4EW |c4c8760d-306c-4ec1-8720-4f9e305c9aed |HR5'_ACE2_chr12L |Knock out ACE2 when used together with HR5'\_ACE2\_chr12R ([BBa\_259HCU8C](https://registry.igem.org/parts/bba-259hcu8c). |Miscellaneous (Deprecated) |300 |2025-09-16T10:13:40.411Z |RFC10 RFC1000 |0 |
|BBa_259HCU8C |e282a210-3a88-4f2b-9176-174767618349 |HR3'_ACE2_chr12R |Knock out ACE2 when used together with HR5'\_ACE2\_chr12L ([BBa\_25MIG4EW](https://registry.igem.org/parts/bba-25mig4ew). |Miscellaneous (Deprecated) |504 |2025-09-16T10:15:20.374Z |RFC10 RFC1000 |0 |
|BBa_25FUWBU7 |3fe55b81-02ff-49cd-8500-15494ccef2fe |HR5'_ChrIX |Yeast genome Chromosome IX insertion, when used together with HR3'\_ChrIX ([BBa\_25PDQZ8Z](https://registry.igem.org/parts/bba-25pdqz8z)). |Miscellaneous (Deprecated) |500 |2025-09-16T10:20:29.005Z |RFC10 RFC1000 |0 |
|BBa_25PDQZ8Z |48770582-e442-4219-bc81-52c4f9627640 |HR3'_ChrIX |Yeast genome Chromosome IX insertion, when used together with HR5'\_ChrIX ([BBa\_25FUWBU7](https://registry.igem.org/parts/bba-25fuwbu7)). |Miscellaneous (Deprecated) |588 |2025-09-16T10:21:47.716Z |RFC10 RFC1000 |0 |
|BBa_25DIWZUL |5ccb5019-1dea-408b-88a7-469a93e2f507 |HR5'_ChrXI |Yeast genome Chromosome XI insertion, when used together with HR3'\_ChrXI ([BBa\_257IAIOG](https://registry.igem.org/parts/bba-257iaiog)). |Miscellaneous (Deprecated) |505 |2025-09-16T10:24:35.012Z |RFC10 RFC1000 |0 |
|BBa_257IAIOG |184843bb-a812-4ba9-9370-3aa0996654ba |HR3'_ChrXI |Yeast genome Chromosome XI insertion, when used together with HR5'\_ChrXI ([BBa\_25DIWZUL](https://registry.igem.org/parts/bba-25diwzul)). |Miscellaneous (Deprecated) |510 |2025-09-16T10:25:56.166Z |RFC10 RFC1000 |0 |
|BBa_256S6J1M |cfa0e508-c833-4401-bcef-ee733df4c0c8 |HsDOR |a human GPCR, used by xxxx (year). |Coding |1116 |2025-08-25T15:26:57.071Z |RFC10 RFC1000 |0 |
|BBa_25RCU5CB |79c24fe6-7bfc-4656-9008-4874210f74c8 |DrDHCR7 |Redirect zymosterol to cholesterol, used by xxxx (year). |Coding |1437 |2025-08-25T15:32:36.943Z |RFC10 RFC1000 |0 |
|BBa_25FOVO4C |85e7de01-e774-4298-9eca-fb6f756b16a0 |DrDHCR24 |Redirect zymosterol to cholesterol, used by xxxx (year). |Coding |1551 |2025-08-25T15:39:48.046Z |RFC10 RFC1000 |0 |
|BBa_K3748015 |ce65ca07-1bba-48ff-853a-f6cdacdb8a85 |pREV1 |To be fullfiled |Regulatory |713 |2022-10-09T15:15:25.000Z |RFC10 RFC1000 |0 |
|BBa_251I2TLJ |c0829c6b-ee4e-448d-a41e-a4f1d4f084f8 |pLacD |pLacD is an IPTG-inducible promoter that enables conditional expression in yeast when used together with LacI. It exhibits moderate expression strength upon induction and relatively low basal activity in the absence of IPTG. |Promoter |190 |2025-09-13T09:00:46.538Z |RFC10 RFC1000 |0 |
|BBa_252BO17G |567e67b8-6653-41b6-9385-077eb604c3f5 |pTet2 |pTet2 is a tetracycline-inducible promoter that, in the presence of doxycycline, is activated by rtTA binding to TetO sequences to drive transcription, reaching medium-to-strong expression levels that depend on the rtTA variant and doxycycline concentration. |Promoter |225 |2025-09-13T08:53:36.297Z |RFC10 RFC1000 |0 |
|BBa_K3944000 |ec8b30d1-f1e7-48c7-922a-37cbc8471cda |ScCUP1p |Promoter sequence to which copper-sensing transcription factor native to yeast binds, upon detecting increasing cellular copper concentrations. The native function of this copper-sensing system is to protect the yeast cell from toxic copper concentrations. By using this promoter sequence, we could induce expression of desired genes in addition to protective effects. |Regulatory |306 |2021-09-14T08:11:05.000Z |RFC10 RFC1000 |0 |
|BBa_259DU8YN |3478aec2-6e30-4c19-a294-07b100bf9a39 |LacI |a transcription factor, responsive to isopropyl β-D-1-thiogalactopyranoside (IPTG). |Coding |1119 |2025-08-26T16:12:39.932Z |RFC10 RFC1000 |0 |
|BBa_251T359T |0ef2f736-3937-41ef-9689-842461f45630 | 重复了，要变成draft不公开
|BBa_2533RATE |f225a24c-9400-4a44-a2e7-aaa91de1db08 |HR3'_SOD1_chr10R |Knock out SOD1 (Chromosome ??) when used together with HR5'\_SOD1\_chr10([BBa\_25CLCLXX](https://registry.igem.org/parts/bba-25clclxx)). |Miscellaneous (Deprecated) |546 |2025-09-14T07:33:19.500Z |RFC10 RFC1000 |0 |
|BBa_25U88LU3 |9a8f4c03-b59c-4d58-b5ee-d0e9207bdce2 |HR5'_CAF_chr14L |Knock out CAF (Chromosome ??) when used together with HR3'\_CAF\_chr14R([BBa\_25O0GI56](https://registry.igem.org/parts/bba-25o0gi56)). |Miscellaneous (Deprecated) |508 |2025-09-14T07:32:48.436Z |RFC10 RFC1000 |0 |
|BBa_25O0GI56 |933d73a9-c8b4-469d-8eb2-1756c7097d92 |HR3'_CAF_chr14R |Knock out CAF (Chromosome ??) when used together with HR5'\_CAF\_chr14L([BBa\_25U88LU3](https://registry.igem.org/parts/bba-25u88lu3)). |Miscellaneous (Deprecated) |546 |2025-09-14T07:32:21.623Z |RFC1000 |0 |
|BBa_25TMD7MC |e2e21484-c098-4adf-b0f5-8145734b228b |HR5'_SWI4_chr5L |Knock out SWI4 (Chromosome ??) when used together with HR3'\_SWI4\_chr5([BBa\_255A36IQ](https://registry.igem.org/parts/bba-255a36iq)). |Miscellaneous (Deprecated) |508 |2025-09-14T07:31:50.083Z |RFC10 RFC1000 |0 |
|BBa_255A36IQ |ca18d466-99a8-4a17-8f1a-f34fd286e517 |HR3'_SWI4_chr5R |Knock out SWI4 (Chromosome ??) when used together with HR5'\_SWI4\_chr5([BBa\_25TMD7MC](https://registry.igem.org/parts/bba-25tmd7mc)). |Miscellaneous (Deprecated) |546 |2025-09-14T07:31:19.304Z |RFC1000 |0 |
|BBa_25Q67BOD |e21edec1-3343-4712-8499-8a973065652d |HR5'_FEN2_chr3L |Knock out FEN2 (Chromosome ??) when used together with HR3'\_FEN2\_chr3([BBa\_25TEB42Q](https://registry.igem.org/parts/bba-25teb42q "(BBa_25TEB42Q")). |Miscellaneous (Deprecated) |508 |2025-09-14T07:30:52.329Z |RFC1000 |0 |
|BBa_25TEB42Q |501c9eab-6b31-472e-b2a7-11c8be3bd3a9 |HR3'_FEN2_chr3R |Knock out FEN2 (Chromosome ??) when used together with HR5'\_FEN2\_chr3([BBa\_25Q67BOD](https://registry.igem.org/parts/bba-25q67bod)). |Miscellaneous (Deprecated) |546 |2025-09-14T07:29:57.787Z |RFC1000 |0 |
|BBa_25ATGCHY |6195f67f-3a7f-4631-b775-a6274be7d0d1 |HR5'_DPB3_chr2L |Knock out DPB3 (Chromosome ??) when used together with HR3'\_DPB3\_chr2([BBa\_25Y6BTXZ](https://registry.igem.org/parts/bba-25y6btxz)) |Miscellaneous (Deprecated) |508 |2025-09-14T07:27:27.606Z |RFC10 RFC1000 |0 |
|BBa_25Y6BTXZ |bf5b7b0e-5b2b-469b-85b5-851a084b0023 |HR3'_DPB3_chr2R |Knock out DPB3 (Chromosome ??) when used together with HR5'\_DPB3\_chr2 ([BBa\_25ATGCHY](https://registry.igem.org/parts/bba-25atgchy "HR5'-DPB3_chr2")). |Miscellaneous (Deprecated) |546 |2025-09-14T07:26:41.338Z |RFC1000 |0 |
|BBa_J435253 |ed2c15c9-e382-407c-8046-caf242d6fe09 |ScURA3-marker, Yeast selection marker, complements URA3 auxotrophy |Yeast selection marker, complements URA3 auxotrophy             Citation: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203683/        ORG: Open Yeast Collection, Open Bioeconomy & The FreeGenes Project        URL: https://freegenes.github.io/genes/BBF10K_000474.html |DNA (Deprecated) |1059 |2023-05-08T14:37:09.000Z |RFC10 RFC1000 |0 |
|BBa_J435273 |a6eefa09-3b94-4eb3-a2c2-261d242a614a |ScHIS3-marker, Yeast selection marker, complements HIS3 auxotrophy |Yeast selection marker, complements HIS3 auxotrophy        Source organism: , Target organism:          Citation: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203683/        ORG: Open Yeast Collection, Open Bioeconomy & The FreeGenes Project        URL: https://freegenes.github.io/genes/BBF10K_000468.html |DNA (Deprecated) |1110 |2023-05-08T14:46:25.000Z |RFC10 RFC1000 |0 |
|BBa_J435279 |1e71761c-0ef2-495b-b274-4cb90b0cb4eb |ScTRP1-marker, Yeast selection marker, complements TRP1 auxotrophy |Yeast selection marker, complements TRP1 auxotrophy        Source organism: , Target organism:          Citation: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203683/        ORG: Open Yeast Collection, Open Bioeconomy & The FreeGenes Project        URL: https://freegenes.github.io/genes/BBF10K_000473.html |DNA (Deprecated) |1625 |2023-05-08T14:48:42.000Z |RFC10 RFC1000 |0 |
|BBa_J435215 |fd4a8257-c189-4b76-b2ed-06d545e7c880 |ScMET17 |Complements methionine auxotrophy        Source organism: S. cerevisiae, Target organism: S. cerevisiae         Citation: https://www.ncbi.nlm.nih.gov/pubmed/3325778        ORG: Open Yeast Collection, Open Bioeconomy & The FreeGenes Project        URL: https://freegenes.github.io/genes/BBF10K_000196.html |Coding |1342 |2023-05-08T14:17:21.000Z |RFC10 |0 |
|BBa_J435500 |e4c8aab4-d16f-4692-b795-f240e9bd85f8 |pOpen_v3 Base Vector AmpR ColE1 FreeGenes |pOpen_v3 Base Vector AmpR ColE1 |Plasmid_Backbone |2001 |2023-06-15T10:21:48.000Z |RFC10 RFC1000 |0 |
|BBa_259JX52V |9df0a3ae-29b9-45e0-88e9-d8865102daba |pOST1 |Yeast promoter, used by xxxx (year) |Promoter |559 |2025-09-09T08:04:42.743Z |RFC10 |0 |
|BBa_K3748013 |702738d7-becd-41f1-a092-afe1a58c642d |pRNR2 |To be fullfiled |Regulatory |713 |2021-10-18T04:29:59.000Z |RFC10 |0 |
|BBa_K530004 |075f9d8a-363a-454e-b99d-0d1421e64182 |STM1 Yeast Promoter |STM1 Promoter is used to regulate the expression of genes within the genome of Saccharomyces Cerevisiae or baker's yeast. |Regulatory |500 |2021-09-08T20:27:04.000Z |RFC10 RFC1000 |0 |
|BBa_K3190001 |7d036ddf-d5d0-440d-a494-9c8c72a8c19a |pTDH3: Yeast strong constitutive promoter |The promoter also called GPD promoter regulates the gene for glyceraldehyde 3-phosphate dehydrogenase (GAPDH). GAPDH is found in the cytoplasm and the cell wall. It is involved in glycolysis and gluconeogenesis, where it catalyzes the reaction of glyceraldehyde-3-phosphate to 1,3 bis-phosphoglycerate (and the reverse reaction).      The TDH3 promoter is used to drive the expression of the TPST1 and TPST2 (Tyrosylprotein sulfotransferase 1 and 2), an enzyme necessary for the relative post-translational modifications of LHCGR receptor. The enzyme catalyzes the sulfation reaction of protein tyrosines. |Regulatory |681 |2021-09-08T20:26:20.000Z |RFC10 RFC1000 |0 |
|BBa_K3944010 |c63b100c-b827-47d9-af49-7efa3703856c |rtTA |Tetracycline-recognising protein fragment, altering conformation upon binding. |Coding |615 |2021-10-19T10:19:56.000Z |RFC10 RFC1000 |0 |
|BBa_K5470009 |131f02f1-4e7c-46d2-b407-2fb092a50b4e |GSG-E2A |2A peptides, characterized by a C-terminal ‘-D12(V/I)ExNPGP19-’ motif, drive a translational recoding event, in which two separate proteins (the ‘upstream’ and ‘downstream’ products) are generated from an ORF containing a 2A coding sequence, with a ‘break’ in the polypeptide backbone between the final 2 amino acids, Gly and Pro, of 2A (formally this Pro is the first amino acid of the downstream protein, but is an essential part of active 2A). The 2A reaction has been termed ‘StopGo’ or ‘stop-carry on’ recoding. |Coding |69 |2024-10-02T10:45:36.000Z |RFC10 RFC1000 |0 |
